frontend-react/src/config/endpoints/lookupTables.ts

```diff
@@ -0,0 +1,72 @@
+import { HTTPMethods, RSApiEndpoints, RSEndpoint } from ".";
+
+// the shape used by the frontend client for value sets
+export interface ValueSet {
+    name: string;
+    createdBy: string;
+    createdAt: string;
+    system: string;
+}
+
+// the shape sent down by the API for value sets
+export interface ApiValueSet {
```

Comment:
Are some of these exports _exclusively_ for unit tests?

If so, you could limit their scope by using them in a named export (search for `_exportForTesting`).

Optional suggestion here.

Reply:
I think this is used for testing, but if anything it's more useful as reference - this is the shape that the API actually sends down, which is especially confusing given the unused `created_at` and `created_by` data (which always comes through empty). We do display that metadata, but we take the values from the metadata from the LookupTable since it is not available on the ValueSet. My hope is that someday that data may start coming back filled in, and then we can use this interface!